ecosystem/node-checker/src/evaluators/direct/api/transaction_availability.rs

```diff
@@ -104,88 +75,125 @@ impl Evaluator for TransactionAvailabilityEvaluator {
     /// baseline produced after a delay. We confirm that the transactions are
     /// same by looking at the version.
     async fn evaluate(&self, input: &Self::Input) -> Result<Vec<EvaluationResult>, Self::Error> {
+        let oldest_baseline_version = input
+            .baseline_index_response
```

Comment:
nit: helper function to extract this and below? (Just to avoid duplication)

Reply:
Feels like overkill imo since we need two helpers, one for each ledger version field.